app/src/main/res/layout/activity_root.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's not necessary to define a new layout for rootActivity, you can use the same of LoginActivity